docs/get-started/introduction.md

```diff
@@ -38,7 +39,16 @@ NFD-Worker is a daemon responsible for feature detection. It then communicates
 the information to nfd-master which does the actual node labeling.  One
 instance of nfd-worker is supposed to be running on each node of the cluster,
 
-## Feature discovery
+## NFD-Topology-Updater
+
+NFD-Topology-Updater is a daemon responsible for examining allocated resources
+on a worker node to account for resources available to be allocated to new pod
+on a per-zone basis (where a zone can be a NUMA node). It then communicates the
+information to nfd-master which does the CR creation corresponding to all the
```

Comment:
Suggestion: add name of the CR and/or reference to the *NodeResourceTopology CR* section below